AssemblerLib/MeshComponentMap.h

```diff
@@ -46,7 +46,18 @@ class MeshComponentMap
         ComponentOrder order);
 
     /// Creates a subset of the current mesh component map.
-    /// The order of components is the same as of the current map.
+    /// The components are those from \c components which are not nullptrs.
+    /// The order (BY_LOCATION/BY_NODE) of components is the same as of the current map.
```

Comment:
`BY_NODE` is confusing because such order does not exist.
Would simply keep `BY_LOCATION`.